src/s4c_the_church_rap_battle.tw2

```diff
@@ -0,0 +1,343 @@
+[(live: 1s)[(if: time > 15s)[$dissolve[//I don’t want leftovers, Fuck you show pony
+All I wanna see is your back now show me//(stop:)]]]]
+
+[(live: 1s)[(if: time > 17s)[$dissolve[(if: $steveRapsFirst is true)[I could feel the power coursing through me and it matched the audience’s roar of approval. [[What a perfect time for the results|The Church - Rap battle - Victory player second]].]
+\(else:)[I could feel the power coursing through me and it matched the audience’s roar of approval. [[Time for Steve’s verse. Ugh|The Church - Rap battle - Second verses steve]].]
+(stop:)]]]]
+
+::The Church - Rap battle - Second verses dogs
+I had to respond to what was in my heart, and what was moving the crowd, so I kept rapping about dogs.
+
+[(live: 1s)[(if: time > 1s)[$dissolve[//I wish I had a puppy when I was growing up
+I’d love to have a flopsy, friendly lil mutt//(stop:)]]]]
+[(live: 1s)[(if: time > 3s)[$dissolve[//It’d be huge now, with an empty gut
+Waiting for my command to eat Steve’s nuts//(stop:)]]]]
+
+[(live: 1s)[(if: time > 5s)[$dissolve[//Yo I’m wild for any dog, especially the shih-tzu’s
+Dogs look at Steve like, “You shit outside too dude?”//(stop:)]]]]
+[(live: 1s)[(if: time > 7s)[$dissolve[//I gotta step it back, I’m bein’ kinda rude
+I’d never let my dog look at Steve or eat his food//(stop:)]]]]
+
+[(live: 1s)[(if: time > 9s)[$dissolve[//Listen, I love giant dogs and little dogs
+When I see them, my heart just clogs//(stop:)]]]]
+[(live: 1s)[(if: time > 11s)[$dissolve[//Up with all the loving, I’m almost crying
+But then I gotta laugh cause Steve on stage dying//(stop:)]]]]
+
+[(live: 1s)[(if: time > 13s)[$dissolve[//Shout out to my terriers and their tiny carriers
+Shout out to the bulldogs and the chihuahuas
+You can pull up a seat and sip on mi agua//(stop:)]]]]
+[(live: 1s)[(if: time > 15s)[$dissolve[//
+Shout out to the dachshunds and all the little homies
+Shoutin’ shut the fuck up Steve, you don’t know me//(stop:)]]]]
+
+[(live: 1s)[(if: time > 17s)[$dissolve[The dog crowd had been standing during the whole verse and now they were rushing towards the stage. Steve was looking pretty sweaty, but he said, “Fuck that weird shit. I’m doing my thing,” and [[launched into his next verse|The Church - Rap battle - Victory dog raps]].
+(stop:)]]]]
+
+::The Church - Rap battle - Victory player first
+It was clear that the crowd had been with me from the start. I could read the energy. As Steve’s homies filed out, a fired up group of ‘lettes stormed the stage and started chanting my name. Some of them were new faces, others I knew and had even heard talk shit. I didn’t care. All were welcome in my temple now that I had made it to the top.
+
+Moon Mystic approached Steve and I, holding both our hands, but I didn’t even wait for him. I raised my own hand and screamed, “YOUR CHAMPION [STAGE NAME]!” Moon Mystic laughed and nodded as [[the crowd stomped in unison|The Church - Mountain intro]].
```

Comment:
Looks like this is supposed to be a variable? Also a few lines down, same thing.

Reply:
LOL, this is what I get for inputting text too fast and then always picking the dog victory option when testing the ending. Thanks for catching this!